### PR TITLE
Update repoman and policy configs

### PR DESCRIPTION
**Pull request by adegeo**
## Summary

- Added policy configs for:
  - Empty issue detection
  - Managing **Not Triaged**
  - Issue opened by dependabot: add **dependencies** label.
  - Issue reopened and has **won't fix** label, remove it.
  - Issue closed, remove **in-progress** label.
  - Issue closed by the customer, add **resolved-by-customer** label.
  - Add area labels to PR based on files touched.
- Update repoman
  - Remove configs settings that were migrated to policy.
  - Add area label for svc/subsvc on issue.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [.github/policies/close-issues.yml](https://github.com/dotnet/docs/blob/fac6829863671d15cab533abbe87e326a791d854/.github/policies/close-issues.yml) | [.github/policies/close-issues](https://review.learn.microsoft.com/en-us/dotnet/.github/policies/close-issues?branch=pr-en-us-44036) |
| [.github/policies/label-issues.yml](https://github.com/dotnet/docs/blob/fac6829863671d15cab533abbe87e326a791d854/.github/policies/label-issues.yml) | [.github/policies/label-issues](https://review.learn.microsoft.com/en-us/dotnet/.github/policies/label-issues?branch=pr-en-us-44036) |
| [.github/policies/label-prs-services.yml](https://github.com/dotnet/docs/blob/fac6829863671d15cab533abbe87e326a791d854/.github/policies/label-prs-services.yml) | [.github/policies/label-prs-services](https://review.learn.microsoft.com/en-us/dotnet/.github/policies/label-prs-services?branch=pr-en-us-44036) |
| [.repoman.yml](https://github.com/dotnet/docs/blob/fac6829863671d15cab533abbe87e326a791d854/.repoman.yml) | [.repoman](https://review.learn.microsoft.com/en-us/dotnet/.repoman?branch=pr-en-us-44036) |

### CODE REVIEW

**Review by BillWagner — APPROVED** [review]
This LGTM @adegeo 

Let's :shipit:

File: .github/policies/issueManagement-emptyIssue.yml
Comment (gewarren) [inline .github/policies/issueManagement-emptyIssue.yml:1]: Can we different file names, like empty-issue.yml?

File: .github/policies/issueManagement-notTriaged.yml
Comment (gewarren) [inline .github/policies/issueManagement-notTriaged.yml:1]: This should maybe go in the "label-issues.yml" file instead.

File: .github/policies/issueManagement-emptyIssue.yml
Comment (gewarren) [reply .github/policies/issueManagement-emptyIssue.yml:1]: Or actually just put this in the close-issues.yml file.

File: .github/policies/label-issues.yml [hunk 1]
```diff
@@ -24,6 +24,56 @@ configuration:
...
+                - or:
+                  - isActivitySender:
+                      user: dependabot
+                  - isActivitySender:
+                      user: dependabot[bot]
+              then:
+                - addLabel:
+                   label: dependencies
+
+            - description: Reopen issue so remove "won't fix" label
```
Comment (gewarren) [inline .github/policies/label-issues.yml:41]:
```suggestion
            - description: Remove "won't fix" label when issue is reopened
```

File: .github/policies/label-issues.yml [hunk 2]
```diff
@@ -24,6 +24,56 @@ configuration:
...
+                - isAction: 
+                    action: Closed
+                - hasLabel:
+                    label: 'in-progress'
+                  
+              then:
+                - removeLabel:
+                   label: 'in-progress'
+
+            - description: Issue closed by author, add "resolved-by-customer" label
```
Comment (gewarren) [inline .github/policies/label-issues.yml:65]:
```suggestion
            - description: Add "resolved-by-customer" label when issue closed by author. 
```

File: .github/policies/labelAdded-mapQuest.yml
Comment (gewarren) [inline .github/policies/labelAdded-mapQuest.yml:1]: Add to label-issues.yml.

**Review by gewarren — APPROVED** [review]

**Review by gewarren — APPROVED** [review]

**Comment by adegeo** [comment]
Going to add in the file path checks, so don't merge.

**Pull request merged**